### PR TITLE
initial website setup

### PR DESCRIPTION
**Pull request by jackjsning**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (jackjsning) [inline .gitignore:1]: nice!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+.DS_Store
```
Comment (jackjsning) [inline .gitignore:2]: reminder to add a pr description!

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "prettier": "2.5.1"
+  }
+}
```
Comment (jackjsning) [inline package.json]: nit: eof

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (jackjsning) [inline public/index.html:1]: remove jack_website submodule

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
+</html>
```
Comment (jackjsning) [inline public/index.html]: eof

File: src/About.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import JackImage from "./Alaska.JPG";
+import styled from "@emotion/styled";
+
+const About = (props) => {
```
Comment (jackjsning) [inline src/About.js]: you can remove props

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+import Nav from "./Navigate";
+import Links from "./MediaLinks";
+import Routes from "./routes";
+import { HashRouter as Router } from "react-router-dom";
+
+const App = () => {
```
Comment (jackjsning) [inline src/App.js]: `export const App`

... (import code)
import { App } from "./App";

File: src/Home.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+const Home = () => {
+  return (
+    <div>
+      <h1>Homepage</h1>
+      <div> This is the Homepage.</div>
+    </div>
+  );
+};
+
+export default Home;
```
Comment (jackjsning) [inline src/Home.js]: eof

File: src/MediaLinks.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    <Navi>
+      <a href="https://github.com/jackjning">Github</a>
+      <a href="https://www.linkedin.com/in/jack-ning-466bb6149/">LinkedIn</a>
+    </Navi>
+  );
+}
+
+export default Links;
+
+const Navi = styled.nav`
```
Comment (jackjsning) [inline src/MediaLinks.js]: Nav?

File: src/Home.js [hunk 10]
```diff
@@ -0,0 +1,10 @@
+import {} from "module";
```
Comment (edjcai) [inline src/Home.js]: what does this do?

File: src/MediaLinks.js [hunk 11]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (edjcai) [inline src/MediaLinks.js:1]: actually for some reason people prefer `import * as React from "react";`

File: src/routes.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+    <Routes>
+      <Route exact path="/" element={<Home />} />
+      <Route path="/About" element={<About />} />
+      <Route path="/Resume" element={<Resume />} />
+      <Route path="/Projects" element={<Projects />} />
+    </Routes>
+  );
+};
+
+export default routes;
```
Comment (edjcai) [inline src/routes.js]: let's simplify to `export const Routes = () => {`?

**Review by edjcai — APPROVED** [review]
Nice work!

**Pull request merged**